### PR TITLE
Update script.py

### PR DESCRIPTION
**Pull request by ANKIT9761**
No need to add Course name in script you have to only enter your icloud id and password.Removed the search functionality as we can skip it the website by selecting the course and then clicking next.

### CODE REVIEW

**Comment by rahulchauhan049** [comment]
I like your Idea of skipping the search part. 
There are some merge conflicts, please resolve them so I can merge. Make sure you are up to date with the main branch.

File: script.py [hunk 1]
```diff
@@ -1,73 +1,67 @@
-import time 
-from selenium import webdriver 
+import time
+from selenium import webdriver
 from selenium.webdriver.support.select import Select
 
-driver = webdriver.Chrome()
+driver = webdriver.Chrome('D:/webdriver/chromedriver.exe')
```
Comment (rahulchauhan049) [inline script.py]: This will not work in other Os. You can remove this path

File: script.py [hunk 2]
```diff
@@ -1,73 +1,67 @@
...
 username = driver.find_element_by_xpath('//*[@id="useriid"]')
 password = driver.find_element_by_xpath('//*[@id="actlpass"]')
 
-
-#Change user name and passwords with your icloud username and password
-username.send_keys('username')  ##################################### Change This Text ############################
-password.send_keys('password')  ##################################### Change This Text ############################
+# Change user name and passwords with your icloud username and password
+username.send_keys('17SCSE112003')  ##################################### Change This Text ############################
+password.send_keys('Gourav8211@')  ##################################### Change This Text ############################
```
Comment (rahulchauhan049) [inline script.py]: You should remove this username and password and replace them with a placeholder.

File: script.py [hunk 3]
```diff
@@ -1,73 +1,67 @@
...
 sel = Select(selectClass)
-#change this text
-sel.select_by_visible_text('B.Tech in CSBA 2017-18 IV Sem VII')  ##################################### Change This Text ############################
 
+sel.select_by_index(1)
+
+time.sleep(10)
+#search = driver.find_element_by_class_name('fa-search')
+# search = driver.find_element_by_xpath('/html/body/div[1]/div/div[1]/div[3]/div/div[2]/div/form/div/div[3]/button')
+#search.click()
```
Comment (rahulchauhan049) [inline script.py]: Can you please remove this commented code?

File: script.py [hunk 4]
```diff
@@ -1,73 +1,67 @@
...
 
-time.sleep(1)
-search = driver.find_element_by_class_name('fa-search')
-search.click()
 
-time.sleep(2)
 
 start = driver.find_element_by_class_name('btn-rounded')
+#start = driver.find_element_by_xpath('/html/body/div[1]/div[1]/div/div[3]/div[2]/div[2]/center/input')
```
Comment (rahulchauhan049) [inline script.py]: Remove this commented code also

File: script.py [hunk 5]
```diff
@@ -1,73 +1,67 @@
...
 start.click()
 
-time.sleep(4)
-
+time.sleep(2)
```
Comment (rahulchauhan049) [inline script.py]: Increase time to 5 seconds. This page take more time to load on slow internt

File: script.py (hunk 1 above)
Comment (ANKIT9761) [reply script.py]: ok

File: script.py (hunk 5 above)
Comment (ANKIT9761) [reply script.py]: I have done all the changes thanks for pointing them out.

**Review by viditvarshney — APPROVED** [review]

File: script.py [hunk 6]
```diff
@@ -86,14 +75,14 @@
...
-                
+
         driver.execute_script("window.scroll(0, 0);")
 
         suggestion_boxes = driver.find_elements_by_class_name('suggestion_boxes')
 
         for suggestion_box in suggestion_boxes:
             try:
-                suggestion_box.send_keys(' Teacher')
+                suggestion_box.send_keys('Teacher')
```
Comment (viditvarshney) [inline script.py]: Give Space Before "Teacher" 
the comment will be: " Good Teacher

**Review by viditvarshney — CHANGES_REQUESTED** [review]

**Review by ANKIT9761 — COMMENTED** [review]
Suggestions box content is updated as requested

File: script.py (hunk 6 above)
Comment (ANKIT9761) [reply script.py]: Noted

**Pull request merged**